lib/editing/editor_ace.dart

```diff
@@ -121,6 +121,10 @@ class _AceEditor extends Editor {
   // TODO: Implement completionActive for ace.
   bool get completionActive => false;
 
+  // TODO: Implement completionActivelyInvoked for comid.
+  bool get completionAutoInvoked => false;
+  set completionAutoInvoked(bool value) => false;
```

Comment:
Do you mean `set completionAutoInvoked(bool value) { };` ?
Is this equivalent to `set completionAutoInvoked(bool value) => null;` ?